src/app.js

```diff
@@ -1 +1,36 @@
+/* eslint-disable no-console */
 'use strict';
+
+const readline = require('readline');
+const { generateRandomNumber } = require('./numberGenerator');
+const { calculateBullsAndCows } = require('./bullsAndCowsCalculator');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const secretNumber = generateRandomNumber();
+
+function playGame() {
+  rl.question('Enter your guess: ', (guessedNumber) => {
+    if (guessedNumber.length !== 4 || !/^\d{4}$/.test(guessedNumber)) {
```

Comment:
4 is a 'magic number', so make sense to move it to the constant
check other places